### DISCUSSION TITLE
마인드 플로우? 어떻게 관리할까...

### DISCUSSION THREAD

**Discussion by Seungyeup** [imimtu/Brick-Back#3] [General]
@Just-gomin  깃헙으로도 과거 기록들이 잘 보이면 좋겠는데 URL로 마인드 플로우 기록 남길 방법 없을까유..

**Comment by Just-gomin** [comment]
mindnode 앱에서 자체 포맷 파일로 저장 가능하니까 Brick Doc Repository를 생성하고, 해당 Repository에서 Brick 관련된 문서를 다 저장할까유...?

**Comment by Just-gomin** [comment]
프로젝트 마다 문서 Repository를 다 둘 수는 없으니, IM2_DOC Repository를 하나 만들까유..?

**Reply by Seungyeup** [reply]
@Just-gomin Organization 단에서 wiki는 지원하지 않나봐... 아니면 DOC Repo를 파서 파일자료는 거기서 관리하고, 해당 repo에 wiki 생성해서 간단한 정리만 할까?

**Reply by Seungyeup** [reply]
+ diagrams.net 보니까 깃헙연동도 되는데..??ㄷ

**Reply by Just-gomin** [reply]
오 diagrams.net 좋지 나 저거 많이 썼어
